ansible/roles/bastion-lite/templates/bastion_ssh_config.j2

```diff
@@ -4,7 +4,11 @@ Host ec2* *.internal
 Host *.example.com
 {% endif %}	
   User {{remote_user | default(hostvars.localhost.remote_user) }}
-  IdentityFile ~/.ssh/{{ env_authorized_key }}.pem
+{% if use_own_key|bool %}
+  IdentityFile ~/.ssh/{{env_authorized_key}}.pem
+{% else %}
+  IdentityFile ~/.ssh/{{key_name}}.pem
```

Comment:
This logic seems flipped -no? The default should be to keep the existing behavior - so need to move `{{ key_name }}` to line 8 above (i.e.: if `use_own_key == true`)